app/models/plant_line.rb

```diff
@@ -74,6 +75,12 @@ def self.permitted_params
     ]
   end
 
+  def self.count_columns
+    [
+      'plant_accessions_count'
```

Comment:
Either this, or CoffeeScript's ``baseColumnDefs`` and ``createCounterLink``, as when I have a PL with no accessions, I get a "NULL plant accession" clickable link under the "Related" button:

![spectacle b18623](https://user-images.githubusercontent.com/107565/35686540-c6ef45e2-076c-11e8-949c-4ba21d42d47c.png)

(PLName: Varuna). I should, however, rather get a proper unclickable link, like this:

![spectacle h18623](https://user-images.githubusercontent.com/107565/35686513-b706c1be-076c-11e8-895a-0c6ba8df9a73.png)

When there are >0 PAs associated, the links looks to be correct. I, however, found other oddities in the PA table, like:

![spectacle e18623](https://user-images.githubusercontent.com/107565/35686631-042bdcae-076d-11e8-889d-2e33ae9e2226.png)

PAName: rres_GK090856

The counting migration didn't report any problem for me. Could it be it's some kind of older problem? In production this PA has a correct count of 12 PSUs.

Reply:
After removing my cache it looks like the first problem is gone, but this second one stays. But maybe it's just me. Simply try to check again such specific cases and see if that happens to you as well.

Reply:
I think the PSU count problem is not related, not happened to me. But I will yet make some checks with data dumped from production.